### ISSUE TITLE
Possibility to delete a post - feature request

### ISSUE THREAD

**Issue by kingu-gidora** [ihabunek/toot#54]
Would it be possible for the post command to return some ID that can later be used to delete the post?

**Comment by ihabunek** [comment]
@kingu-gidora It already gives you the ID in the success message which looks like:
```
Toot posted: https://mastodon.social/@ihabunek/100202249801818634
```
100202249801818634 is the status ID.

I'm also adding a delete command so you can delete statuses by running:
```
toot delete 100202249801818634
```

BTW, I have been thinking about adding an option which would return machine-readable data instead of human readable text. For example, when posting to return the status data in JSON as received from mastodon. But this would need to be implemented for other commands as well and sounds like a lot of work.

**Comment by ihabunek** [comment]
`toot delete` has been implemented and released in v0.19.0. Status ID also added to `toot timeline`.
